Marlin/temperature.h

```diff
@@ -303,6 +303,37 @@ class Temperature {
       #endif
     }
 
+    #if ENABLED(BABYSTEPPING)
+
+      FORCE_INLINE void babystep_axis(AxisEnum axis, int distance) {
```

Comment:
I tried building with the old code, the new code, with and without `FORCE_INLINE`, and they all compiled to the same exact size.